.github/workflows/publish-store.yml

```diff
@@ -1,18 +1,161 @@
+          run: |
+            mkdir -p $HOME/secrets
+            az storage blob download --account-name $ACCOUNT_NAME --container-name $CONTAINER_NAME --name $FILE \
+                --file $HOME/secrets/$FILE --output none
+
+        - name: Retrieve certificates
+          run: |
+            mkdir -p $HOME/certificates
+            az keyvault secret show --id https://bitwarden-ci.vault.azure.net/certificates/ios-distribution |
+              jq -r .value | base64 -d > $HOME/certificates/ios-distribution.p12
+
+        - name: Download Fastlane credentials
+          env:
+            ACCOUNT_NAME: bitwardenci
+            CONTAINER_NAME: mobile
+            FILE: appstoreconnect-fastlane.json
+          run: |
+            mkdir -p $HOME/secrets
+            az storage blob download --account-name $ACCOUNT_NAME --container-name $CONTAINER_NAME --name $FILE \
+              --file $HOME/secrets/$FILE --output none
+
+        - name: Configure Keychain Access
+          env:
+            KEYCHAIN_PASSWORD: ${{ secrets.IOS_KEYCHAIN_PASSWORD }}
+          run: |
+            security create-keychain -p $KEYCHAIN_PASSWORD build.keychain
+            security default-keychain -s build.keychain
+            security unlock-keychain -p $KEYCHAIN_PASSWORD build.keychain
+            security set-keychain-settings -lut 1200 build.keychain
+
+            security import $HOME/certificates/ios-distribution.p12 -k build.keychain -P "" -T /usr/bin/codesign \
+              -T /usr/bin/security
+            security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k $KEYCHAIN_PASSWORD build.keychain
+
+        - name: Update beta Fastlane Appfile
+          run: |
+            echo 'app_identifier "com.8bit.bitwarden.beta"' > fastlane/Appfile
+
+        - name: Set up private auth key
+          run: |
+            mkdir ~/private_keys
+            cat << EOF > ~/private_keys/AuthKey_J46C83CB96.p8
+            ${{ secrets.APP_STORE_CONNECT_AUTH_KEY }}
+            EOF
+
+        - name: Upload app to with Fastlane
+          env:
+            VERSION_CODE: ${{ inputs.build-number }}
+            VERSION_NAME: ${{inputs.build-version}}
+            PHASED_RELEASE: ${{ inputs.phased-release }}
+            PRODUCT: ${{ inputs.product }}
+            CHANGELOG: ${{ inputs.changelog }}
+            FASTLANE_ENABLE_BETA_DELIVER_SYNC_SCREENSHOTS: true
+            SUBMIT_RELEASE: ${{ inputs.submit-release }}
+          run: |
+            if [ "$PRODUCT" = "Password Manager" ]; then
+              PACKAGE_NAME="com.8bit.bitwarden"
+              METADATA_PATH="metadata"
+              echo "product is Password Manager"
+            elif [ "$PRODUCT" = "Authenticator" ]; then
+              PACKAGE_NAME="com.bitwarden.authenticator"
+              METADATA_PATH="authenticator_metadata"
+            else
+              echo "Unsupported product: $PRODUCT"
+              exit 1
+            fi
+
+            bundle exec fastlane submit_release \
```

Comment:
⛏️  Maybe rename this fastlane name so it's not conflated with the workflow input?